package.json

```diff
@@ -83,6 +83,7 @@
     "eslint-plugin-sonarjs": "^0.5.0",
```

Comment:
I don't actually get much performance benefit from this, because the copying seems to run in parallel with "generate lazy", which still takes the same amount of time:

On master:
```
[46:11.6]  copy started ...
[46:11.6]  generate lazy started ...
[46:33.9]  copy finished (5138 files) in 22.33 s
[46:46.4]  generate lazy finished in 34.83 s
```

On this branch:
```
[06:53.5]  copy started ...
[06:53.5]  generate lazy started ...
[06:53.9]  copy finished (13 files) in 436 ms
[07:26.3]  generate lazy finished in 32.80 s
```

Reply:
That ☝️ in itself wouldn't be a problem, since this still brings a benefit on Linux, but:

After trying around a bit more, it seems I have to run npm install every time I stopped the dev-server with ctrl-c, otherwise the node-modules are missing, and I also have to remove the build output folders before running `npm start` again, because otherwise the build process seems to just end up in some endless loop, spinning up my computer fan to maximum… 😢

Reply:
:dizzy_face: omg that seems like a real hassle haha :smile: 
I can put this back in draft, it seems like getting the tests to work was a real hassle as well and don't want to spend several more hours on it right now... I have tried getting the mocking to work, but it seems to be really hard to get right when running in puppeteer :thinking:  
Even something like this did not seem to work :disappointed: 
```ts
    await page.addScriptTag({
        content: `
            window.fetch = () => {
                return Promise.resolve({
                    text: () => Promise.resolve('${emptySvg}')
                });
            };
        `,
    });
```
It was worth a try at least! But if it is only marginally faster there is no point in doing this. I just looked at the `copy finished (5138 files) in 22.33 s` when testing this, I should have clocked it as well :smile:

Reply:
😞

But yeah, keep it as a draft. It would be really nice if we can improve performance a bit. It would also be really nice to have the different versions of the published docs use the same copy of limel-icons8, instead of having one copy for each version, and fixing that seems like it might be similar to fixing this?

Btw, you are linking the whole node_modules folder in this PR. Have you tried only linking the folder where the icons reside?

Reply:
Yeah I linked the whole node_modules folder since that was the fastest and easiest way :upside_down_face: It would not be that hard to make the rollup plugin take a config of what to create links to instead